app/controllers/application_controller.rb

```diff
@@ -47,7 +49,7 @@ def require_user
   end
 
   def require_oauth
-    @oauth = @user.access_token(OAUTH_KEY) if @user && defined? OAUTH_KEY
+    @oauth = @user.access_token(OAUTH_KEY) if current_user && defined? OAUTH_KEY
```

Comment:
You missed one in this line ;-)

As far as I could tell last night that was the only one in any view or controller that had been missed, which is probably pretty good doing.